kfs-purap/src/main/java/edu/arizona/kfs/module/purap/document/web/struts/PaymentRequestForm.java

```diff
@@ -0,0 +1,32 @@
+package edu.arizona.kfs.module.purap.document.web.struts;
+
+import org.kuali.kfs.module.purap.document.PaymentRequestDocument;
```

Comment:
Would we want to import the edu.arizona.kfs.module.purap.document.PaymentRequestDocument (since it extends the baseline file) here to pull in any necessary overrides that may be needed now or in the future i.e. PaymentMethodCode?